apps/sms/views/conversation/js/conversation.js

```diff
@@ -642,10 +642,20 @@ var ConversationView = {
       args.draftId ? this.handleDraft(+args.draftId) : Promise.resolve();
 
     return draftPromise.then(() => {
-      if (args.focusComposer) {
-        Compose.focus();
-      } else {
-        this.recipients.focus();
+      switch (args.focus) {
+        case 'composer':
+          Compose.focus();
+          break;
+        case 'recipients':
+          this.recipients.focus();
+          break;
+        default:
+          if (this.hasValidRecipients()) {
```

Comment:
right now we don't save invalid recipients, but we will eventually. Wondering what we should do if we have only invalid recipients... I'd say... we'll see when it will happen :p